packages/subapp-web/lib/init.js

```diff
@@ -34,12 +30,32 @@ module.exports = function setup(setupContext) {
     basePath: ""
   };
 
+  let inlineRuntimeJS = "";
+  let rtJsCdnEntryPoint;
+  if (process.env.NODE_ENV === "production") {
+    rtJsCdnEntryPoint = Object.keys(cdnJsBundles).filter(ep =>
+      cdnJsBundles[ep].includes("runtime.bundle")
+    )[0];
+    const runtimeFullPath =
+      rtJsCdnEntryPoint &&
+      cdnJsBundles[rtJsCdnEntryPoint] &&
+      Path.join(Path.resolve("dist"), "js", Path.basename(cdnJsBundles[rtJsCdnEntryPoint]));
+    if (Fs.existsSync(runtimeFullPath)) {
+      inlineRuntimeJS = Fs.readFileSync(runtimeFullPath).toString();
+    }
+  }
+
   const webSubAppJs = `<script id="bundleAssets" type="application/json">
 ${JSON.stringify(bundleAssets)}
 </script>
 <script>/*LJ*/${loadJs}/*LJ*/
+/*rt*/${inlineRuntimeJS} /*rt*/
 ${clientJs}
 ${cdnJs}
+${
+  inlineRuntimeJS && rtJsCdnEntryPoint && `window.xarcV1.markBundlesLoaded([${rtJsCdnEntryPoint}]);`
```

Comment:
please make this code and line 52 more concise.

Reply:
You are right. see updated PR.
![image](https://user-images.githubusercontent.com/2591536/83206865-b19db580-a106-11ea-945d-03cbe4eb42d4.png)
![image](https://user-images.githubusercontent.com/2591536/83206884-c11cfe80-a106-11ea-97ab-69b9ce41eb14.png)